source/_integrations/uvc.markdown

```diff
@@ -14,14 +15,11 @@ The platform connects to the Unifi NVR software and automatically discovers/adds
 
 ### Setup
 
-It is recommended that you create a new user for this platform in the NVR software and only give the user the permissions it need to operate.
-
-- The API key is found in `User` -> `My account` -> `API Access` in the NVR software.
-- The camera password is found in `Settings` -> `Camera Settings` -> `Camera Password` in the NVR software.
+It is recommended that you create a new user for this platform in the NVR software and only give the user the permissions it need to operate. The API key is found in `User` -> `My account` -> `API Access` in the NVR software.
 
 ### Configuration
 
-To enable, add the following to your `configuration.yaml` file:
+To enable, either add via the Integrations UI, or add the following to your `configuration.yaml` file:
```

Comment:
Could you please explain to the user how? Like a click path? Is there a difference in functionality between the two options that need to be explained?

For a new user to our platform, this is a bit limited and unguided.

Reply:
I would rather add here reference to common instruction of adding integrations via the config flow instead of putting it in every separated component. But fine

Reply:
I agree, but until we have that, we have to add it manually.

Reply:
Yep. Is the sentence fine, or to messy?